features/share/access-controls/user-access-controls/README.md

```diff
@@ -0,0 +1,265 @@
+<p align="center">
+  <a href="https://vendia.net/">
+    <img src="https://raw.githubusercontent.com/vendia/examples/main/vendia-logo.png" alt="vendia logo" width="100px">
+  </a>
+</p>
+
+# User Access Controls
+User access controls protect the user account management operations provided by Vendia Share.  This prevents unauthorized access to user management functions and enables features like self-service user management within an organization.
+
+Authorization to specific user account management operations that act on user accounts is granted through a [Role Based Access Control (RBAC) mechanism](https://www.vendia.net/docs/share/rbac).  Before going further, it's important to define a few key RBAC terms, as they relate to user access controls.
+
+## Key Terminology
```

Comment:
I found this section very dense. I will make a few suggestions. Trying to balance between 'how precise' and 'how simple'.

Reply:
Great call.  I simplified the language in the User and Uni sections, both here and in the blog post.